client/src/pages/CampaignShow.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useCallback, useState } from 'react';
+import Button from "@material-ui/core/Button";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import CampaignStep from "../components/CampaignStep";
+import { makeStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography"
+
+export default function CampaignShow() {
+  const [steps, addStep] = useState([])
+  const [contacted, setContacted] = useState(35)
+  const [opened, setOpened] = useState(30)
+  const [replied, setReplied] = useState(3)
+  const classes = useStyles()
+
+  // const createData = (contacted, opened, replied) => {
```

Comment:
Great catch. I had it there in case we needed multiple rows for the campaigns table. I'm not anticipating that we will, so I'll go ahead and remove that.